lib/messages/create.js

```diff
@@ -22,7 +51,11 @@ module.exports = message({
       },
       request: {
         Buffer: buffer,
-        DesiredAccess: 0x001701df,
+        /**
+         * https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-smb2/77b36d0f-6016-458a-a7a0-0f4a72ae1534
+         * DesiredAccess is a construction of : SYNCHRONIZE, DELETE, READ_CONTROL, WRITE_DAC, FILE_WRITE_ATTRIBUTES, FILE_READ_ATTRIBUTES, FILE_DELETE_CHILD, FILE_WRITE_EA, FILE_READ_DATA, FILE_WRITE_DATA, FILE_APPEND_DATA, FILE_READ_EA
+         */
```

Comment:
You can remove this comment, it's no longer necessary now that the value is explicit.